enatega-multivendor-admin/src/constraints/constraints.js

```diff
@@ -74,6 +74,10 @@ const constraints = {
   username: {
     presence: {
       allowEmpty: false
```

Comment:
You can add a message here as well in allowEmpty check and then use the message based on whatever check is failed to print in the above component.